### ISSUE TITLE
Undefined index: status trying to mute a participant

### ISSUE THREAD

**Issue by vkulov** [twilio/twilio-php#408]
We are creating a conference and trying to mute a participant but get a strange error about 'status' missing in response payload.

According to REST docs 'status' is not available in json rest responses:
{
	"account_sid": "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
	"call_sid": "CA386025c9bf5d6052a1d1ea42b4d16662",
	"conference_sid": "CFbbe46ff1274e283f7e3ac1df0072ab39",
	"date_created": "Wed, 18 Aug 2010 20:20:10 +0000",
	"date_updated": "Wed, 18 Aug 2010 20:23:06 +0000",
	"end_conference_on_exit": true,
	"muted": true,
	"hold": false,
	"start_conference_on_enter": true,
	"uri": "/2010-04-01/Accounts/ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/Conferences/CFbbe46ff1274e283f7e3ac1df0072ab39/Participants/CA386025c9bf5d6052a1d1ea42b4d16662.json"
}

**Version:**
5.5

### Code Snippet
```php
$participant = $client
    ->conferences("CFbbe4632a3c49700934481addd5ce1659")
    ->participants("CA386025c9bf5d6052a1d1ea42b4d16662")
    ->update(['mute'=>true]);

```

### Exception / Log
```
ErrorException: Undefined index: status
/twilio/sdk/Twilio/Rest/Api/V2010/Account/Conference/ParticipantInstance.php
line 57

```

### Steps to Reproduce
1. Create a conference
2. Mute a participant

**Comment by vkulov** [comment]
We get the same error 
Undefined index: status
doing this call:
$this->twilioClient
                    ->account
                    ->conferences('XXX')
                    ->participants
                    ->read();

**Comment by vkulov** [comment]
We found that this line 
`            'status' => $payload['status'],
`
is introduced in twilio sdk 5.5

However in the 'status' is not being returned in json REST responses. Is there something we have missed when upgrading to sdk 5.5?
I tried the API explorer and indeed 'status' is not being returned in any of the REST JSON responses.

**Comment by efossier** [comment]
@vkulov That is indeed a bug, thanks for bringing that to our attention! We'll have a fix out soon.

**Comment by efossier** [comment]
@vkulov This should be fixed in 5.6.0 https://github.com/twilio/twilio-php/blob/master/CHANGES.md#2017-03-03-version-560
